pkg/elb/cleanup.go

```diff
@@ -93,15 +128,21 @@ func Cleanup(ctx context.Context, ec2API ec2iface.EC2API, elbAPI elbiface.ELBAPI
 			}
 			logger.Debug("load balancer %s and its security groups were deleted by the cloud provider", name)
 			// The load balancer and its security groups have been deleted
-			delete(loadBalancers, name)
+			delete(awsLoadBalancers, name)
 		}
 	}
-
-	if len(loadBalancers) > 0 {
-		return fmt.Errorf("deadline surpased waiting for load balancers to be deleted")
+	numLB := len(awsLoadBalancers)
+	if numLB > 0 {
+		lbs := make([]string, numLB)
```

Comment:
You could just use `append` in the below loop if you use `make([[]string, 0, numLB)` or just get `i` from `for i, name := range..` (or use `0, numLB` and `i` togegther) or just skip `make` completely and use `append`.
Pick one of those :slightly_smiling_face:

Reply:
I can't take `i` from `range` because it is `map`, but I can rewrite to `append` 🙂

Reply:
ah right didn't see that, thanks!